mirrord/cli/src/connection.rs

```diff
@@ -83,6 +83,8 @@ where
         let k8s_api = KubernetesAPI::create(config)
             .await
             .map_err(CliError::KubernetesApiFailed)?;
+
+        k8s_api.detect_openshift().await?;
```

Comment:
We shouldn't exit on this error - just log in info level that we failed trying to detect.

Reply:
we should also do it only if we don't use operator.